business-wiki-frontend/pages/home/index.tsx

```diff
@@ -0,0 +1,91 @@
+import defaultStyles from "../../shared/styles/Default.module.scss";
+import buttonStyles from "../../shared/styles/Button.module.scss";
+import moduleStyles from "../../shared/styles/Home.module.scss";
+
+import Head from "next/head";
+import Link from "next/link";
+import Footer from "../../shared/components/Footer";
+import React from "react";
+import { UserContext } from "../../shared/contexts/UserContext";
+import { SidepanelListElementInterface } from "../../shared/models/SidepanelListElementInterface";
+import { Page } from "../../shared/models/Page";
+import SidepanelListElement from "../../shared/components/SidepanelListElement";
+import SidepanelListAddElement from "../../shared/components/SidepanelListAddElement";
+import PageForm from "../../shared/components/PageForm";
+import { UserInterface } from "../../shared/models/UserInterface";
+import PageDetails from "../../shared/components/PageDetails";
+
+export default function Home() {
+  const context = React.useContext(UserContext);
+
+  let pagesList: Array<Page> = [
+    {
+      id: 0,
+      title: "Welcome Page",
+      content: "This is the greeting page created by BusinessWiki for you.",
+      createdBy: {
+        id: 1,
+        username: "Business Wiki",
+        password: "1234"
+      } as UserInterface,
+    } as Page
+  ];
+
+  let sidepanelPagesList: Array<SidepanelListElementInterface> = [
+    {
+      id: 0,
+      title: "Welcome Page",
+      clickEvent: (e) => showPageContent(e)
+    } as SidepanelListElementInterface
+  ];
+
+  const showPageContent = (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+    return (
+      <></>
+    );
+  };
+
+  const addPageEvent = () => {
+    return (
+      <PageForm />
+    );
+  };
+
+  const showSidepanelList = (): JSX.Element => {
+    return (
+      <>
+        {sidepanelPagesList.length > 0 ? sidepanelPagesList.map((element) => <SidepanelListElement key={element.id} value={element} />) : ""}
+      </>
+    );
+  };
+
+  return (
+    <div className={defaultStyles.container}>
```

Comment:
defaultStyles i moduleStyles też można zdestrukturyzować.

Reply:
Resolved in c1ee6c5f030d621c244b5972aa8521733146b9e5